cranelift-codegen/meta/src/shared/legalize.rs

```diff
@@ -272,6 +289,59 @@ pub(crate) fn define(insts: &InstructionGroup, imm: &Immediates) -> TransformGro
         ],
     );
 
+    narrow.custom_legalize(icmp_imm, "narrow_icmp_imm");
+
+    let intcc_eq = Literal::enumerator_for(&imm.intcc, "eq");
```

Comment:
Now that there's the custom legalization, should these legalizations be dropped instead?

Reply:
The custom legalization is for icmp_imm, these are for icmp.

Reply:
@bjorn3 on spot :) I already dropped the legalizations for `icmp_imm`.